client/blocks/edit-gravatar/test/index.jsx

```diff
@@ -42,13 +42,14 @@ describe( 'EditGravatar', function() {
 		};
 	} );
 
-	before( function() {
+	before( function( done ) {
```

Comment:
I still occasionally get the timeout error:

`Error: timeout of 2000ms exceeded. Ensure the done() callback is being called in this test.`

It may be the case that we're sometimes running into an error when running `before()`, but it's hard to reproduce, so haven't caught it yet with a try-catch block.